subsys/net/lib/dns/resolve.c

```diff
@@ -914,6 +927,20 @@ int dns_resolve_close(struct dns_resolve_context *ctx)
 
 	for (i = 0; i < SERVER_COUNT; i++) {
 		if (ctx->servers[i].net_ctx) {
+			struct net_if *iface;
+
+			iface = net_context_get_iface(ctx->servers[i].net_ctx);
+
+#if defined(CONFIG_NET_MGMT_EVENT_INFO)
```

Comment:
Can't you use IS_ENABLED() here? (if not, it probably needs dummies in net_mgmt.h)

Reply:
Good point, fixed.